frontend/src/features/leases/detail/LeasePages/surplus/Surplus.tsx

```diff
@@ -0,0 +1,85 @@
+import { FormSection } from 'components/common/form/styles';
+import { ColumnWithProps, Table } from 'components/Table';
+import { getIn, useFormikContext } from 'formik';
+import { ILease, IProperty } from 'interfaces';
+import { prettyFormatDate } from 'utils';
+
+interface IDeclaration {
+  id: number;
+  identifier: string;
+  declarationType: string;
+  date: string;
+  comments: string;
+}
+
+const columns: ColumnWithProps<IDeclaration>[] = [
+  {
+    Header: 'PID / Identifier',
+    accessor: 'identifier',
+    maxWidth: 40,
+  },
+  {
+    Header: 'Surplus Declaration',
+    accessor: 'declarationType',
+    maxWidth: 40,
+    Cell: ({ row: { original } }) => {
+      const declarationType = original.declarationType;
+      const isYesDeclarationType = declarationType.toUpperCase() === 'YES';
+      return (
+        <>
+          {isYesDeclarationType ? (
+            <strong className="text-success">{declarationType}</strong>
+          ) : (
+            <span>{declarationType}</span>
+          )}
+        </>
+      );
+    },
+  },
+  {
+    Header: 'Declaration Date',
+    accessor: 'date',
+    Cell: ({ cell: { value } }) => prettyFormatDate(value),
+    maxWidth: 50,
+  },
+  {
+    Header: 'Declaration Comments',
+    accessor: 'comments',
+    minWidth: 150,
+  },
+];
+
+const Surplus: React.FunctionComponent<{}> = () => {
+  const { values } = useFormikContext<ILease>();
+  const organizations: IProperty[] = getIn(values, 'properties') ?? [];
```

Comment:
/nit: sorry just noticed this. var name seems off - shouldn't it be **properties** instead?